docker/scripts/setup_env

```diff
@@ -28,11 +28,17 @@ if [[ -n "${MYSQL_PORT_3306_TCP_ADDR}" || ("${DATABASE_ADAPTER}" == "mysql2" &&
     DATABASE_HOST=${DATABASE_HOST:-${MYSQL_PORT_3306_TCP_ADDR}}
     DATABASE_PORT=${DATABASE_PORT:-${MYSQL_PORT_3306_TCP_PORT:-3306}}
     DATABASE_ENCODING=${DATABASE_ENCODING:-utf8mb4}
+    DATABASE_NAME=${DATABASE_NAME:-${MYSQL_DATABASE}}
```

Comment:
Are those new variable names needed? We could also put `DATABASE_NAME` and `mysql.env` and `postgres.env` couldn't we?

Reply:
The `mysql` and `postgres` images use these variables to setup their database. In the old `docker-compose.yml` files, the same data were entered twice for the different containers (e.g., under `MYSQL_PASSWORD` and `HUGINN_DATABASE_PASSWORD`). Yes, we could do the same in `mysql.env`, but I feel that now with the `env_file` option, it makes more sense to keep it DRY.

Reply:
I did not realize it was part of the old `docker-compose` file, I think we can get rid of those env variables in the compose file. For postgres I know that the image works without giving it a database name. In case of mysql the docs say

> This variable is optional and allows you to specify the name of a database to be created on image startup.

I think we don't want this. The container might create the database with the wrong encoding/collation and we normally handle the database creation. Can you test if the compose files work for mysql without passing `MYSQL_DATABASE`?